app/models/metadata_presenter/page_answers.rb

```diff
@@ -75,7 +74,16 @@ def raw_date_answer(component_id)
     def filename(path)
       return sanitize(path) if path.nil?
 
-      sanitize(path).gsub(/&gt;/, '').gsub(/&lt;/, '').delete('>"[]{}*?:|]/<').delete('\\')
+      filename = sanitize(path).gsub(/&gt;/, '').gsub(/&lt;/, '').delete('>"[]{}*?:|]/<').delete('\\')
+
+      if count.presence && count.positive?
```

Comment:
```suggestion
      if count.present? && count.positive?
```

Similar to above comment.